Changes.md

```diff
@@ -19,6 +19,7 @@ API
 ---
 
 - Spreadsheet : Added an `addColumn()` overload with an `adoptEnabledPlug` boolean argument. This allows cells to reuse the `enabled` plug from their `value` plug if it has one.
+- SpreadsheetUI : Added `formatValue()` and `registerValueFormatter()` methods to support custom formatting for extension plug types.
```

Comment:
Just a quick note as a reminder about out chat re: supporting some visual feedback of colors/etc... don't know if you want to address that in this PR or a future one.

Reply:
I have it on my todo list, but will deal with it in a follow up PR.